docs/cudf/source/cudf_polars/index.rst

```diff
@@ -9,6 +9,12 @@ and run on the CPU.
 
 Benchmark
 ---------
+
+````{note}
+Following benchmarks were performed with `POLARS_GPU_ENABLE_CUDA_MANAGED_MEMORY` environment variable set to `"0"`.
```

Comment:
```suggestion
The following benchmarks were performed with `POLARS_GPU_ENABLE_CUDA_MANAGED_MEMORY` environment variable set to `"0"`. Using managed memory (the default) imposes a performance cost in order to avoid out of memory errors. Peak performance can still be attained by setting the environment variable to 1.
```